### PR TITLE
4th History Remarks

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Made Kael'thas content, just and shrewd.
- Added 2 lore characters to **Marris** dynasty: **Thereas** and **Stephon Marris**.
- Reworked **Sylvanas** and **Nathanos** characters. They're nice and kind before the Scourge came and are cruel after.
- Nathanos is in Anasterian's court and is Sylvanas' lover.

# How to test:
Check the changes in-game and tell your opinion.

### CODE REVIEW

**Comment by zumbak04** [comment]
> However, one major problem is that Sylvanas is using her undead portrait while she is alive, which is definitely misleading.

Yes, noticed it too. It's a bug from the master, so I fixed it on a separate branch, here: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/fd0d81b87164863cfc76174c537d37743d5eb8b8.

File: history/characters/42000_high_elf.txt [hunk 1]
```diff
@@ -53,19 +53,30 @@
 	properties="ai0e0a00000000"
 	culture=high_elf religion=cult_of_sunwell
 	martial=7 diplomacy=6 stewardship=6 intrigue=7 learning=8
-	add_trait=mastermind_theologian add_trait=ambitious add_trait=proud add_trait=stubborn add_trait=wroth add_trait=cynical
+	add_trait=mastermind_theologian
+	add_trait=content # Don't want to rule the elves
+	add_trait=just # Though he is not worthy to be the next king
```
Comment (MemeWizard) [inline history/characters/42000_high_elf.txt]: Are you sure that "though he is not worthy to be the next king" is reason for giving "just" trait, not "depressed"?

Comment (zumbak04) [reply history/characters/42000_high_elf.txt]: He got depressed on the Frozen Throne branch.

Comment (MemeWizard) [reply history/characters/42000_high_elf.txt]: So... What you meant to say?
That you replaced just with depressed on the Frozen Throne branch or that he is both just and depressed on the Frozen Throne branch?

Comment (zumbak04) [reply history/characters/42000_high_elf.txt]: Both.

**Pull request merged**